### PR TITLE
feat: adding a getContentType method into the OAS Operation class

### PR DESCRIPTION
**Pull request by erunion**
This method exists currently in `@readme/oas-to-har` but it makes more sense for it to live here.

https://github.com/readmeio/api-explorer/blob/master/packages/oas-to-har/src/index.js#L32-L53

### CODE REVIEW

File: packages/tooling/__tests__/operation.test.js [hunk 1]
```diff
@@ -4,10 +4,16 @@ const petstore = require('./__fixtures__/petstore.json');
 const multipleSecurities = require('./__fixtures__/multiple-securities.json');
 const referenceSpec = require('./__fixtures__/local-link.json');
 
+describe('#getContentType', () => {
+  it('should return the content type on an operation', () => {
```
Comment (domharrington) [inline packages/tooling/__tests__/operation.test.js:8]: Are there any other tests in oas-to-har that we can pull over? Few little weird edge cases in that function! 

https://github.com/readmeio/api-explorer/blob/06aa341b629680b255e1469e427c76fe08e05bc3/packages/oas-to-har/__tests__/index.test.js#L1325-L1470

Comment (erunion) [reply packages/tooling/__tests__/operation.test.js:8]: There were a couple I had missed.
